src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/MSSQL/0028-View-SectionDim-for-Student-Engagement.sql

```diff
@@ -0,0 +1,103 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'SectionDim')
+BEGIN
+    DROP VIEW analytics.SectionDim
+END
+GO
+
```

Comment:
The comment on the DS2 version of this file applies equally to the changes you see here.